### PR TITLE
Rebalance of Melee Weapons

### PR DESCRIPTION
**Pull request by Aexolott**
## About the PR

Rebalances spears, shivs, and the captain's sabre.

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Cody ~ Aexxie
- tweak: Rebalanced crude melee weapons and the Captain's Sabre.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/knife.yml [hunk 1]
```diff
@@ -101,12 +101,16 @@
...
+    tags:
+      - CombatKnife
+      - Knife
   - type: Sprite
     sprite: Objects/Weapons/Melee/survival_knife.rsi
     size: 2
     state: icon
   - type: Item
-    size: 10
+    size: 12
```
Comment (Slava0135) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/knife.yml:113]: wtf why you changed size

**Comment by Slava0135** [comment]
balance changes must have a reason to make, and you say nothing why you changed anything and what changed exactly

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml [hunk 2]
```diff
@@ -22,6 +22,10 @@
     tags:
     - CaptainSabre
   - type: DisarmMalus
+  - type: Tool
```
Comment (Slava0135) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml:25]: sabre can pry doors wtf

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml [hunk 3]
```diff
@@ -22,6 +22,10 @@
...
+    qualities:
+      - Prying
```
[the remark this answers is not in the document]
Comment (Aexolott) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml:27]: Slowed by more than half

**Comment by Aexolott** [comment]
> The weapon changes don't really address the issues that we talked about in vc. I thought the solution was going to be adding strong pierce resistance to most structures, or is this unrelated to that?

Miscommunication, but explained in dms.

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/knife.yml (hunk 1 above)
Comment (Aexolott) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/knife.yml:113]: I don't particularly remember changing the size, but this was over a week ago. It was likely to compensate for the buff the knife was given.

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml (hunk 2 above)
Comment (Aexolott) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml:25]: QoL improvement so captain doesn't have to seek out a crowbar all of the time. It functions similarly to the fireaxe's prying ability. It's speed has been lowered significantly. It is still better to have a crowbar, but the sword will work if there is no other options.

**Comment by asperger-sind** [comment]
I think this balance patch is unnecessary and just makes more balancing problems.
Looking at the spear, 12 pierce damage if wielded with 1 attack speed and a weight value of 95 and having the cool feature of using spears as a throwing weapon thrown into the gutter with 8 pierce damage on throw, slightly better than throwing a steel tile for reference measure.

Combat knifes being nearly as good as the Captain's sabre does not make any sense, only 2 slash damage worse with everything else being the same stats while not being an overly rare item.

The survival knife having more weight is weird, what was wrong with it having 10 weight value?
On the topic of that, why does replacing a glass shard with a uranium glass shard in your spear suddenly make it's weight value go down by 70 (from 95 to 25)

If anything, I'm already against nerfing weapons with piercing damage, we have way too much pierce resistances on way too much gear, so having spears deal less damage is ehh.

If you argue that spears can have chem injection and use it as a point to nerf the base damage value of the spears, just nerf the chem injection instead, there is no reason for a melee weapon to be balanced around a niche feature and not vice-versa.

**Comment by Aexolott** [comment]
> I think this balance patch is unnecessary and just makes more balancing problems. Looking at the spear, 12 pierce damage if wielded with 1 attack speed and a weight value of 95 and having the cool feature of using spears as a throwing weapon thrown into the gutter with 8 pierce damage on throw, slightly better than throwing a steel tile for reference measure.
> 
> Combat knifes being nearly as good as the Captain's sabre does not make any sense, only 2 slash damage worse with everything else being the same stats while not being an overly rare item.
> 
> The survival knife having more weight is weird, what was wrong with it having 10 weight value? On the topic of that, why does replacing a glass shard with a uranium glass shard in your spear suddenly make it's weight value go down by 70 (from 95 to 25)
> 
> If anything, I'm already against nerfing weapons with piercing damage, we have way too much pierce resistances on way too much gear, so having spears deal less damage is ehh.
> 
> If you argue that spears can have chem injection and use it as a point to nerf the base damage value of the spears, just nerf the chem injection instead, there is no reason for a melee weapon to be balanced around a niche feature and not vice-versa.

I will probably look over it again if need be, but as it is the crude spears are just as powerful as the sabre.

**Comment by Aexolott** [comment]
> > I will probably look over it again if need be, but as it is the crude spears are just as powerful as the sabre.
> 
> Not really? You're comparing pierce damage - being one of the worse damage types due to resistances - with 1 attack speed that is also a wieldable to a rare item with 1.5 attack speed and slash damage, being the best damage type in the game.
> 
> If anything I'm fine with spears being slightly worse at melee attacks, but make them a good option for throwing, possibly also decreasing the minimum size, so the item is less of a straight melee weapon and more of a hybrid between melee and throwable ranged.

I wasn't intending to nerf throwing, and for the record changing spear weight was **not** my change. It was someone else's pr (iirc it was emogarbage).

**Comment by TaralGit** [comment]
Why not have reinforced windows spawn ordinary glass shards, eliminate reinforced shards/shivs/spears and rebalance the damage of the remaining weapons?

I originally wanted to get rid of reinforced shards but I didn't know how and didn't wanted to mess the code.

**Pull request closed without merging**